# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/alsa-utils-lintl.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- alsa-utils-1.0.14/seq/aseqnet/Makefile.am.orig	2007-05-31 10:07:21.000000000 +0200
+++ alsa-utils-1.0.14/seq/aseqnet/Makefile.am	2007-10-09 09:27:57.000000000 +0200
@@ -3,4 +3,5 @@
 
 bin_PROGRAMS = aseqnet
 aseqnet_SOURCES = aseqnet.c
+aseqnet_LDADD = -lintl
 man_MANS = aseqnet.1
